packages/front-end/pages/attributes.tsx

```diff
@@ -37,6 +37,11 @@ const FeatureAttributesPage = (): React.ReactElement => {
       <td className="text-gray">
         {v.datatype}
         {v.datatype === "enum" && <>: ({v.enum})</>}
+        {v.format && v.format !== "none" && (
+          <p className="my-0">
+            <small>(format: {v.format})</small>
+          </p>
+        )}
```

Comment:
If any of the attributes are specifying the version string format, I think we should add a notice/warning banner to this page:

> Version string attributes are only supported in the latest Javascript and React SDK versions.  Other languages are coming soon.

Open to other ideas to accomplish the same thing.  Maybe in the attribute form itself instead?

Reply:
ok, added cf4d3fdab4b53c56c2acebcc6d0c49417a8afb26

<img width="832" alt="image" src="https://github.com/growthbook/growthbook/assets/113377031/06ba8520-6e2e-4145-8189-b91236773fc5">